tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -1487,7 +1489,7 @@ def _impl(ctx):
             unfiltered_compile_flags_feature,
             treat_warnings_as_errors_feature,
             archive_param_file_feature,
-        ] + layering_check_features(ctx.attr.compiler)
+        ] + layering_check_features(ctx.attr.compiler, False)
```

Comment:
I've thought a bit how to avoid the passing of an uncommented boolean flag literal here.
I think the best would be just to call with a named param: layering_check_features(ctx.attr.compiler, is_macos=False)
wdyt?
Alternatives I've considered:
One could pass in the added flag list instead, but even if generating it with macos/linux named methods this would break the nice locality of the workaround, and the method names would become hideous.
One could pass in is_linux from the outer if statement; then at least the purpose of the bool flag would be clear, but inside layering_check_features() is_macos is more expressive than is_linux.

Reply:
did the named arg